### PR TITLE
Locally sourced assets

### PR DESCRIPTION
**Pull request by PaulOstazeski**
#### What does this PR do?
* Makes local copies of jquery and google fonts, serving them from localhost instead of making requests out to google.
#### Where should the reviewers start?
* The main _change_ is to the sources for loading jquery and google fonts.
* The rest is the addition of the font definition files.
#### How should this be manually tested?
* Open up a browser console to the network activity tab. Watch the content of the "Domain" column.
* Navigate around the site.
* Do you see any requests to google or gstatic for jquery or fonts?
#### Any background context you want to provide?
This is the most naive implementation I could think of.
#### What are the relevant tickets?
#39 
#### Screenshots?
N/A
#### Questions:
1. Do we need non-latin fonts?
1. Is https://github.com/jekyll/jekyll-assets a better solution?

### CODE REVIEW

File: fonts/google/-L14Jk06m6pUHB-5mXQQnYX0hVgzZQUfRDuZrPvH3D8.woff2 [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```
Comment (alimi) [inline fonts/google/-L14Jk06m6pUHB-5mXQQnYX0hVgzZQUfRDuZrPvH3D8.woff2]: There are a few font files here that kinda look like 404 HTML pages. Maybe these fonts didn't download right?

whats-up-with-that

**Comment by PaulOstazeski** [comment]
D'oh

**Comment by PaulOstazeski** [comment]
I think I've corrected that.

```console
% file assets/fonts/*
assets/fonts/0eC6fl06luXEYWpBSJvXCIX0hVgzZQUfRDuZrPvH3D8.woff2:                      Web Open Font Format (Version 2), TrueType, length 10824, version 2.0
assets/fonts/2tsd397wLxj96qwHyNIkxHYhjbSpvc47ee6xR_80Hnw.woff2:                      Web Open Font Format (Version 2), TrueType, length 10156, version 2.0
assets/fonts/77FXFjRbGzN4aCrSFhlh3oX0hVgzZQUfRDuZrPvH3D8.woff2:                      Web Open Font Format (Version 2), TrueType, length 10712, version 2.0
assets/fonts/97uahxiqZRoncBaCEI3aW4X0hVgzZQUfRDuZrPvH3D8.woff2:                      Web Open Font Format (Version 2), TrueType, length 7940, version 2.0
assets/fonts/aZMswpodYeVhtRvuABJWvBkAz4rYn47Zy2rvigWQf6w.woff2:                      Web Open Font Format (Version 2), TrueType, length 5084, version 2.0
assets/fonts/d-6IYplOFocCacKzxwXSOJBw1xU1rKptJj_0jans920.woff2:                      Web Open Font Format (Version 2), TrueType, length 10276, version 2.0
assets/fonts/e7MeVAyvogMqFwwl61PKhBkAz4rYn47Zy2rvigWQf6w.woff2:                      Web Open Font Format (Version 2), TrueType, length 7900, version 2.0
assets/fonts/Fl4y0QdOxyyTHEGMXX8kcYX0hVgzZQUfRDuZrPvH3D8.woff2:                      Web Open Font Format (Version 2), TrueType, length 6832, version 2.0
assets/fonts/frNV30OaYdlFRtH2VnZZdhkAz4rYn47Zy2rvigWQf6w.woff2:                      Web Open Font Format (Version 2), TrueType, length 6656, version 2.0
assets/fonts/gwVJDERN2Amz39wrSoZ7FxkAz4rYn47Zy2rvigWQf6w.woff2:                      Web Open Font Format (Version 2), TrueType, length 764, version 2.0
assets/fonts/Hgo13k-tfSpn0qi1SFdUfZBw1xU1rKptJj_0jans920.woff2:                      Web Open Font Format (Version 2), TrueType, length 10324, version 2.0
assets/fonts/I3S1wsgSg9YCurV6PUkTOYX0hVgzZQUfRDuZrPvH3D8.woff2:                      Web Open Font Format (Version 2), TrueType, length 5176, version 2.0
assets/fonts/isZ-wbCXNKAbnjo6_TwHToX0hVgzZQUfRDuZrPvH3D8.woff2:                      Web Open Font Format (Version 2), TrueType, length 7044, version 2.0
assets/fonts/jSN2CGVDbcVyCnfJfjSdfIX0hVgzZQUfRDuZrPvH3D8.woff2:                      Web Open Font Format (Version 2), TrueType, length 5020, version 2.0
assets/fonts/-L14Jk06m6pUHB-5mXQQnYX0hVgzZQUfRDuZrPvH3D8.woff2:                      Web Open Font Format (Version 2), TrueType, length 788, version 2.0
assets/fonts/mbmhprMH69Zi6eEPBYVFhYX0hVgzZQUfRDuZrPvH3D8.woff2:                      Web Open Font Format (Version 2), TrueType, length 3016, version 2.0
assets/fonts/mx9Uck6uB63VIKFYnEMXrYX0hVgzZQUfRDuZrPvH3D8.woff2:                      Web Open Font Format (Version 2), TrueType, length 5024, version 2.0
assets/fonts/NYDWBdD4gIq26G5XYbHsFIX0hVgzZQUfRDuZrPvH3D8.woff2:                      Web Open Font Format (Version 2), TrueType, length 3036, version 2.0
assets/fonts/ODelI1aHBYDBqgeIAH2zlCxe5Tewm2_XWfbGchcXw4g.woff2:                      Web Open Font Format (Version 2), TrueType, length 3508, version 1.3276
assets/fonts/ODelI1aHBYDBqgeIAH2zlIa1YDtoarzwSXxTHggEXMw.woff2:                      Web Open Font Format (Version 2), TrueType, length 9836, version 1.3276
assets/fonts/ODelI1aHBYDBqgeIAH2zlJbPFduIYtoLzwST68uhz_Y.woff2:                      Web Open Font Format (Version 2), TrueType, length 11400, version 1.3276
assets/fonts/oHi30kwQWvpCWqAhzHcCSIX0hVgzZQUfRDuZrPvH3D8.woff2:                      Web Open Font Format (Version 2), TrueType, length 6960, version 2.0
assets/fonts/oOeFwZNlrTefzLYmlVV1UIX0hVgzZQUfRDuZrPvH3D8.woff2:                      Web Open Font Format (Version 2), TrueType, length 7900, version 2.0
assets/fonts/Pru33qjShpZSmG3z6VYwnYX0hVgzZQUfRDuZrPvH3D8.woff2:                      Web Open Font Format (Version 2), TrueType, length 8040, version 2.0
assets/fonts/PwZc-YbIL414wB9rB1IAPYX0hVgzZQUfRDuZrPvH3D8.woff2:                      Web Open Font Format (Version 2), TrueType, length 3024, version 2.0
assets/fonts/rGvHdJnr2l75qb0YND9NyIX0hVgzZQUfRDuZrPvH3D8.woff2:                      Web Open Font Format (Version 2), TrueType, length 788, version 2.0
assets/fonts/RxZJdnzeo3R5zSexge8UUZBw1xU1rKptJj_0jans920.woff2:                      Web Open Font Format (Version 2), TrueType, length 10248, version 2.0
assets/fonts/toadOcfmlt9b38dHJxOBGCD5K6T8I4oZ1X3Xvlj_UeP3rGVtsTkPsbDajuO5ueQw.woff2: Web Open Font Format (Version 2), TrueType, length 3556, version 1.3276
assets/fonts/toadOcfmlt9b38dHJxOBGCP2LEk6lMzYsRqr3dHFImA.woff2:                      Web Open Font Format (Version 2), TrueType, length 11448, version 1.3276
assets/fonts/toadOcfmlt9b38dHJxOBGDOFnJNygIkrHciC8BWzbCz3rGVtsTkPsbDajuO5ueQw.woff2: Web Open Font Format (Version 2), TrueType, length 9844, version 1.3276
assets/fonts/toadOcfmlt9b38dHJxOBGJkF8H8ye47wsfpWywda8og.woff2:                      Web Open Font Format (Version 2), TrueType, length 11044, version 1.3276
assets/fonts/toadOcfmlt9b38dHJxOBGKyGJhAh-RE0BxGcd_izyev3rGVtsTkPsbDajuO5ueQw.woff2: Web Open Font Format (Version 2), TrueType, length 9740, version 1.3276
assets/fonts/toadOcfmlt9b38dHJxOBGMms7UHsIbjUxEJqIwog-i_3rGVtsTkPsbDajuO5ueQw.woff2: Web Open Font Format (Version 2), TrueType, length 3428, version 1.3276
assets/fonts/toadOcfmlt9b38dHJxOBGMzFoXZ-Kj537nB_-9jJhlA.woff2:                      Web Open Font Format (Version 2), TrueType, length 11256, version 1.3276
assets/fonts/toadOcfmlt9b38dHJxOBGMZXFz2iDKd7GJNSaxRYiSj3rGVtsTkPsbDajuO5ueQw.woff2: Web Open Font Format (Version 2), TrueType, length 3480, version 1.3276
assets/fonts/toadOcfmlt9b38dHJxOBGO4s1Ux4PuImWPk5fSr6HPL3rGVtsTkPsbDajuO5ueQw.woff2: Web Open Font Format (Version 2), TrueType, length 9584, version 1.3276
assets/fonts/ty9dfvLAziwdqQ2dHoyjphkAz4rYn47Zy2rvigWQf6w.woff2:                      Web Open Font Format (Version 2), TrueType, length 10512, version 2.0
assets/fonts/UX6i4JxQDm3fVTc1CPuwqoX0hVgzZQUfRDuZrPvH3D8.woff2:                      Web Open Font Format (Version 2), TrueType, length 784, version 2.0
assets/fonts/VvXUGKZXbHtX_S_VCTLpGhkAz4rYn47Zy2rvigWQf6w.woff2:                      Web Open Font Format (Version 2), TrueType, length 2940, version 2.0
assets/fonts/ZLqKeelYbATG60EpZBSDy4X0hVgzZQUfRDuZrPvH3D8.woff2:                      Web Open Font Format (Version 2), TrueType, length 10656, version 2.0
```

**Comment by latazzajones** [comment]
Hey Paul - this is looking great but I'm getting this error on `jekyll build`screen shot 2017-01-16 at 9 43 08 am

**Comment by PaulOstazeski** [comment]
Yep, that's because I didn't add it to git. I repeat: D'oh!

**Pull request merged**